tests/e2e/tests/e2e/GitSsh.spec.ts

```diff
@@ -72,19 +75,24 @@ suite('Git with ssh workflow', async () => {
         await editor.waitText('Untitled-0', 'ssh-rsa');
     });
 
-
     test('Add a SSH key to GitHub side and clone by ssh link', async () => {
         const sshName: string = WorkspaceNameHandler.generateWorkspaceName('test-SSH-', 5);
         const publicSshKey = await cheGitAPI.getPublicSSHKey();
         await gitHubUtils.addPublicSshKeyToUserAccount(TestConstants.TS_GITHUB_TEST_REPO_ACCESS_TOKEN, sshName, publicSshKey);
         await cloneTestRepo();
-
+        // workaround for issue: https://github.com/eclipse/che/issues/19101
```

Comment:
FYI: the problem should be fixed within https://github.com/eclipse/che/issues/19544, so the actual issue for tracking now is https://github.com/eclipse/che/issues/19544.

Reply:
Thanks, replaced.